src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -294,6 +320,8 @@ class FilterConsensusReads
       val depths2 = rec[Array[Short]](BaRawReadCount)
       val errors1 = rec[Array[Short]](AbRawReadErrors)
       val errors2 = rec[Array[Short]](BaRawReadErrors)
+      val bases1  = rec.get[String](AbConsensusBases).getOrElse("N" * rec.length)
```

Comment:
This is nasty.  Is there really no better way to handle this than manufacturing a string of Ns per each read if this is missing?